Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -321,6 +321,16 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         var (line, column) = position
         let string = textView.string
```

Comment:
Maybe rename this to "text" or "content". Calling a variable "string" is a bit weird imo

Reply:
Agreed

Reply:
It looks like that name was already there. Might be out of scope for this PR

Reply:
Resolving this because the name is outside of scope here, eventually this block should be changed to not reference the raw textView string at all but that is for a future PR and for the most part won't affect the changes made here.